files/build_templates/buffers_config.j2

```diff
@@ -50,6 +49,9 @@ def
                 {%- set neighbor_role = neighbor.type %}
                 {%- if 'asic' == neighbor_role | lower %}
                          {%- set roles1 = 'internal' %}
+                         {%- if 'internal' not in ports2cable %}
+                             {%- set _ = ports2cable.update({'internal': '5m'}) %}
+                         {%- endif -%}
```

Comment:
Prefer not to have here. Which specific test case is failing.

Reply:
Discussed with Abhishek,  devices that override ports2cable will have this problem. Fix looks good.